components/movementsensor/adxl345/adxl345_test.go

```diff
@@ -33,7 +33,9 @@ func TestInterrupts(t *testing.T) {
 	i2cHandle.CloseFunc = func() error { return nil }
 	i2cHandle.WriteByteDataFunc = func(context.Context, byte, byte) error { return nil }
 	i2cHandle.ReadByteDataFunc = func(context.Context, byte) (byte, error) { return byte(1<<6 + 1<<2), nil }
-	i2cHandle.ReadBlockDataFunc = func(context.Context, byte, uint8) ([]byte, error) { return []byte{byte(0xE5)}, nil }
+	i2cHandle.ReadBlockDataFunc = func(context.Context, byte, uint8) ([]byte, error) {
+		return []byte{byte(0xE5), byte(0x1), byte(0x2), byte(0x3), byte(0x4), byte(0x5), byte(0x6)}, nil
```

Comment:
Maybe put in a comment noting that this might return _more_ data than we're expecting, but leave as-is if you think such a comment wouldn't be useful. 

We call this in 2 different places: when initializing the component, we request 1 byte and expect `[0xE5]` to be returned (but we only examine the first byte even if more are returned, which is why this PR is okay even if it's slightly surprising), and then in the goroutine we request 6 bytes and expect 6 bytes to be returned (which is why the old version had problems that this version fixes).

Reply:
added!